python/triton/language/core.py

```diff
@@ -254,6 +255,8 @@ def __init__(self, element_ty: dtype, shape: List):
         self.numel = 1
         for s in self.shape:
             self.numel *= s
+            if self.numel > TRITON_MAX_TENSOR_NUMEL:
```

Comment:
shouldn't this be after the `for`, and also shouldn't the error message include the actual value of `self.numel` ?